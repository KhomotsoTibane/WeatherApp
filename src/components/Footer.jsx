import React from 'react'

function Footer() {
    const date= new Date();
    let year = date.getFullYear();
    
  return (
  <footer>
        <span className="footer">Â© khomotso Tibane {year}</span>
    </footer>
  )
}

export default Footer