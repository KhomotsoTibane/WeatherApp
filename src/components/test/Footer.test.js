import React from "react";
import Footer from "../Footer";
import renderer from "react-test-renderer";



test("renders correctly without error",() => {
    const tree = renderer.create(<Footer/>).toJSON();
    expect(tree).toMatchSnapshot();
})

