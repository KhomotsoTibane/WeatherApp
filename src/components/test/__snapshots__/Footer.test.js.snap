// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly without error 1`] = `
<footer>
  <span
    className="footer"
  >
    Â© khomotso Tibane 
    2022
  </span>
</footer>
`;
